app/views/vacancies/index.html.slim

```diff
@@ -44,3 +43,5 @@
         - else
           = render "vacancies/search/no_results"
       = govuk_pagination(pagy: @pagy)
+      - if @vacancies.any?
+        = render "vacancies/search/sort_and_stats", form: @form, vacancies_search: @vacancies_search, vacancies: @vacancies
```

Comment:
will the options to sort vacancies appear at the bottom of the page?

Reply:
they shouldnt, i didnt pay enough attention, sort should be top stats bottom, will change